src/main/java/it/cnr/isti/hpc/wikipedia/parser/ArticleParser.java

```diff
@@ -538,4 +526,116 @@ private void setDisambiguation(Article a) {
 		}
 	}
 
+	/*
+	 * returns a pair <Namespace, TopicId>
+	 */
+	public static org.apache.commons.math3.util.Pair<String, String> extractNETopic(String target)
```

Comment:
Why the two different types of Pair? They seem very similar, are both really needed?